tests/foreman/api/test_contentviewfilter.py

```diff
@@ -57,3 +81,265 @@ def test_get_with_bad_args(self):
+        gen_string('utf8', randint(1, 85)),
+        gen_string('html', randint(1, 85)),
+    )
+    def test_create_pck_group_cvf_with_different_names(self, name):
+        """Test: Create new package group content filter using different inputs
+        as a name
+
+        @Assert: Content view filter created successfully and has correct name
+        and type
+
+        @Feature: Content View Filter - Create
+
+        """
+        cvf = entities.PackageGroupContentViewFilter(
+            content_view=self.content_view,
+            name=name,
+        ).create()
+        self.assertEqual(cvf.name, name)
+        self.assertEqual(cvf.type, 'package_group')
+
+    @data(
+        gen_string('alphanumeric', randint(1, 255)),
+        gen_string('alpha', randint(1, 255)),
+        gen_string('cjk', randint(1, 85)),
+        gen_string('latin1', randint(1, 255)),
+        gen_string('numeric', randint(1, 255)),
+        gen_string('utf8', randint(1, 85)),
+        gen_string('html', randint(1, 85)),
+    )
+    def test_create_rpm_cvf_with_different_names(self, name):
+        """Test: Create new RPM content filter using different inputs as
+        a name
+
+        @Assert: Content view filter created successfully and has correct name
+        and type
+
+        @Feature: Content View Filter - Create
+
+        """
+        cvf = entities.RPMContentViewFilter(
+            content_view=self.content_view,
+            name=name,
+        ).create()
+        self.assertEqual(cvf.name, name)
+        self.assertEqual(cvf.type, 'rpm')
+
+    @data(True, False)
+    def test_create_cvf_with_inclusion(self, inclusion):
+        """Test: Create new content view filter with different inclusion values
+
+        @Assert: Content view filter created successfully and has correct
+        inclusion value
+
+        @Feature: Content View Filter - Create
+
+        """
+        cvf = entities.RPMContentViewFilter(
+            content_view=self.content_view,
+            inclusion=inclusion,
+            name=self.cvf_name,
+        ).create()
+        self.assertEqual(cvf.inclusion, inclusion)
+
+    @data(
+        gen_string('alphanumeric', randint(1, 255)),
+        gen_string('alpha', randint(1, 255)),
+        gen_string('cjk', randint(1, 85)),
+        gen_string('latin1', randint(1, 255)),
+        gen_string('numeric', randint(1, 255)),
+        gen_string('utf8', randint(1, 85)),
+        gen_string('html', randint(1, 85)),
+    )
+    def test_create_cvf_with_description(self, description):
+        """Test: Create new content filter using different inputs as a
+        description
+
+        @Assert: Content view filter created successfully and has correct
+        description
+
+        @Feature: Content View Filter - Create
+
+        """
+        cvf = entities.RPMContentViewFilter(
+            content_view=self.content_view,
+            description=description,
+            name=self.cvf_name,
```

Comment:
Is there any reason to pass in an explicit name here? If a name is not supplied, the `create_missing` function will do so for you.